docs/routing/route_configuration_options.md

```diff
@@ -1,87 +1,87 @@
 # Route configuration options
```

Comment:
Pls extract these changes to another PR. Try to have standalone PRs for every change/fix you do